src/components/InvoicesList/InvoicesList.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import { Loading } from '../Loading/Loading';
+import { InjectAppServices } from '../../services/pure-di';
+import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { FormattedMessage } from 'react-intl';
+
+const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
+  const [stateInvoices, setStateInvoices] = useState({ loading: true });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setStateInvoices({ loading: true });
+      const invoicesResponse = await dopplerBillingApiClient.getInvoices(0, 0);
+      if (!invoicesResponse.success) {
+        setStateInvoices({ loading: false, items: [] });
```

Comment:
Warning!, when there is an error, it says that "there are no invoices to show", I think that it is not true.

Reply:
When the service throws an error, what is the expected behavior?

Reply:
Maybe you could show a similar message but saying that there was an error? Probably @GusBaamonde has a special component to show errors, but for the moment I think that simply say "There was an error gathering the invoices" should be fine.

Reply:
Perfect, thanks @andresmoschini